// @esri/calcite-components/dist/cjs/en-ca-fbbc6759.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.5.1
 */
'use strict';

const calciteInputTimePicker_calciteTimePicker_entry = require('./calcite-input-time-picker.calcite-time-picker-aaf25b2c.js');
require('./index-53b1ff8d.js');
require('./form-2df08523.js');
require('./dom-df1b15c9.js');
require('./guid-554ae9c1.js');
require('./resources-c2b4175e.js');
require('./interactive-73fea2f1.js');
require('./browser-32d4fc79.js');
require('./key-7cd54fd8.js');
require('./label-7df0b58c.js');
require('./component-6d0ec09f.js');
require('./loadable-3f828fdc.js');
require('./locale-a12dc490.js');
require('./observers-1ac50ff9.js');
require('./focusTrapComponent-920dd05b.js');
require('./math-df1a3f90.js');
require('./t9n-c17f3b75.js');
require('./openCloseComponent-ab283b2a.js');
require('./Validation-0575c46f.js');

// English (Canada) [en-ca]
var locale = {
  name: 'en-ca',
  weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split('_'),
  months: 'January_February_March_April_May_June_July_August_September_October_November_December'.split('_'),
  weekdaysShort: 'Sun_Mon_Tue_Wed_Thu_Fri_Sat'.split('_'),
  monthsShort: 'Jan_Feb_Mar_Apr_May_Jun_Jul_Aug_Sep_Oct_Nov_Dec'.split('_'),
  weekdaysMin: 'Su_Mo_Tu_We_Th_Fr_Sa'.split('_'),
  ordinal: function ordinal(n) {
    return n;
  },
  formats: {
    LT: 'h:mm A',
    LTS: 'h:mm:ss A',
    L: 'YYYY-MM-DD',
    LL: 'MMMM D, YYYY',
    LLL: 'MMMM D, YYYY h:mm A',
    LLLL: 'dddd, MMMM D, YYYY h:mm A'
  },
  relativeTime: {
    future: 'in %s',
    past: '%s ago',
    s: 'a few seconds',
    m: 'a minute',
    mm: '%d minutes',
    h: 'an hour',
    hh: '%d hours',
    d: 'a day',
    dd: '%d days',
    M: 'a month',
    MM: '%d months',
    y: 'a year',
    yy: '%d years'
  }
};
calciteInputTimePicker_calciteTimePicker_entry.dayjs.locale(locale, null, true);

exports.default = locale;
