// @esri/calcite-components/dist/cjs/fr-ch-eeb2045e.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.5.1
 */
'use strict';

const calciteInputTimePicker_calciteTimePicker_entry = require('./calcite-input-time-picker.calcite-time-picker-aaf25b2c.js');
require('./index-53b1ff8d.js');
require('./form-2df08523.js');
require('./dom-df1b15c9.js');
require('./guid-554ae9c1.js');
require('./resources-c2b4175e.js');
require('./interactive-73fea2f1.js');
require('./browser-32d4fc79.js');
require('./key-7cd54fd8.js');
require('./label-7df0b58c.js');
require('./component-6d0ec09f.js');
require('./loadable-3f828fdc.js');
require('./locale-a12dc490.js');
require('./observers-1ac50ff9.js');
require('./focusTrapComponent-920dd05b.js');
require('./math-df1a3f90.js');
require('./t9n-c17f3b75.js');
require('./openCloseComponent-ab283b2a.js');
require('./Validation-0575c46f.js');

// French (Switzerland) [fr-ch]
var locale = {
  name: 'fr-ch',
  weekdays: 'dimanche_lundi_mardi_mercredi_jeudi_vendredi_samedi'.split('_'),
  months: 'janvier_février_mars_avril_mai_juin_juillet_août_septembre_octobre_novembre_décembre'.split('_'),
  weekStart: 1,
  weekdaysShort: 'dim._lun._mar._mer._jeu._ven._sam.'.split('_'),
  monthsShort: 'janv._févr._mars_avr._mai_juin_juil._août_sept._oct._nov._déc.'.split('_'),
  weekdaysMin: 'di_lu_ma_me_je_ve_sa'.split('_'),
  ordinal: function ordinal(n) {
    return n;
  },
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD.MM.YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY HH:mm',
    LLLL: 'dddd D MMMM YYYY HH:mm'
  },
  relativeTime: {
    future: 'dans %s',
    past: 'il y a %s',
    s: 'quelques secondes',
    m: 'une minute',
    mm: '%d minutes',
    h: 'une heure',
    hh: '%d heures',
    d: 'un jour',
    dd: '%d jours',
    M: 'un mois',
    MM: '%d mois',
    y: 'un an',
    yy: '%d ans'
  }
};
calciteInputTimePicker_calciteTimePicker_entry.dayjs.locale(locale, null, true);

exports.default = locale;
