// @esri/calcite-components/dist/esm/it-f6184b31.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.5.1
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-9df95509.js';
import './index-a8ffce66.js';
import './form-48bdc344.js';
import './dom-a024c014.js';
import './guid-cc244ce6.js';
import './resources-5f694c3a.js';
import './interactive-8ec04fb9.js';
import './browser-c0bb0374.js';
import './key-412bf44e.js';
import './label-f76c866a.js';
import './component-c278e912.js';
import './loadable-f0bba14f.js';
import './locale-be98a84a.js';
import './observers-4cc2b52c.js';
import './focusTrapComponent-f8ae5c2d.js';
import './math-0f85cb27.js';
import './t9n-eb41bc7f.js';
import './openCloseComponent-9520e134.js';
import './Validation-b481231d.js';

// Italian [it]
var locale = {
  name: 'it',
  weekdays: 'domenica_lunedì_martedì_mercoledì_giovedì_venerdì_sabato'.split('_'),
  weekdaysShort: 'dom_lun_mar_mer_gio_ven_sab'.split('_'),
  weekdaysMin: 'do_lu_ma_me_gi_ve_sa'.split('_'),
  months: 'gennaio_febbraio_marzo_aprile_maggio_giugno_luglio_agosto_settembre_ottobre_novembre_dicembre'.split('_'),
  weekStart: 1,
  monthsShort: 'gen_feb_mar_apr_mag_giu_lug_ago_set_ott_nov_dic'.split('_'),
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD/MM/YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY HH:mm',
    LLLL: 'dddd D MMMM YYYY HH:mm'
  },
  relativeTime: {
    future: 'tra %s',
    past: '%s fa',
    s: 'qualche secondo',
    m: 'un minuto',
    mm: '%d minuti',
    h: 'un\' ora',
    hh: '%d ore',
    d: 'un giorno',
    dd: '%d giorni',
    M: 'un mese',
    MM: '%d mesi',
    y: 'un anno',
    yy: '%d anni'
  },
  ordinal: function ordinal(n) {
    return n + "\xBA";
  }
};
dayjs.locale(locale, null, true);

export default locale;
