// @esri/calcite-components/dist/esm/lv-36c5a2d3.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.5.1
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-9df95509.js';
import './index-a8ffce66.js';
import './form-48bdc344.js';
import './dom-a024c014.js';
import './guid-cc244ce6.js';
import './resources-5f694c3a.js';
import './interactive-8ec04fb9.js';
import './browser-c0bb0374.js';
import './key-412bf44e.js';
import './label-f76c866a.js';
import './component-c278e912.js';
import './loadable-f0bba14f.js';
import './locale-be98a84a.js';
import './observers-4cc2b52c.js';
import './focusTrapComponent-f8ae5c2d.js';
import './math-0f85cb27.js';
import './t9n-eb41bc7f.js';
import './openCloseComponent-9520e134.js';
import './Validation-b481231d.js';

// Latvian [lv]
var locale = {
  name: 'lv',
  weekdays: 'svētdiena_pirmdiena_otrdiena_trešdiena_ceturtdiena_piektdiena_sestdiena'.split('_'),
  months: 'janvāris_februāris_marts_aprīlis_maijs_jūnijs_jūlijs_augusts_septembris_oktobris_novembris_decembris'.split('_'),
  weekStart: 1,
  weekdaysShort: 'Sv_P_O_T_C_Pk_S'.split('_'),
  monthsShort: 'jan_feb_mar_apr_mai_jūn_jūl_aug_sep_okt_nov_dec'.split('_'),
  weekdaysMin: 'Sv_P_O_T_C_Pk_S'.split('_'),
  ordinal: function ordinal(n) {
    return n;
  },
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD.MM.YYYY.',
    LL: 'YYYY. [gada] D. MMMM',
    LLL: 'YYYY. [gada] D. MMMM, HH:mm',
    LLLL: 'YYYY. [gada] D. MMMM, dddd, HH:mm'
  },
  relativeTime: {
    future: 'pēc %s',
    past: 'pirms %s',
    s: 'dažām sekundēm',
    m: 'minūtes',
    mm: '%d minūtēm',
    h: 'stundas',
    hh: '%d stundām',
    d: 'dienas',
    dd: '%d dienām',
    M: 'mēneša',
    MM: '%d mēnešiem',
    y: 'gada',
    yy: '%d gadiem'
  }
};
dayjs.locale(locale, null, true);

export default locale;
